lib/ansible/modules/windows/win_firewall_profile.ps1

```diff
@@ -0,0 +1,118 @@
+#!powershell
+
+# Copyright : (c) 2019, Simon Lecoq <simon.lecoq@live.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#AnsibleRequires -CSharpUtil Ansible.Basic
+#Requires -Module Ansible.ModuleUtils.CamelConversion
+
+# Specs and init
+$spec = @{
+    options = @{
+        profiles = @{ type = "list"; elements = "str"; choices = "Domain", "Private", "Public"; default = @( "Domain", "Private", "Public" ); aliases = @( "profile" ) }
```

Comment:
Choices should be in the `snake_case` format unless there is a good reason to preserve the original names. The same applies to all the other options you have as well.

Reply:
Thanks, for reviewing. I will apply your recommandations as soon as possible.